developer-docs/latest/installation/cli.md

```diff
@@ -8,7 +8,7 @@ Fast-track local install for getting Strapi running on your computer.
 
 #### Node.js
 
-Strapi only requires [Node.js](https://nodejs.org). The current recommended version to run Strapi is **Node v14**.
+Strapi only requires [Node.js](https://nodejs.org). The current supported release of Node is **Node LTS v14**, other versions of Node may not be compatible with the current release of Strapi.
```

Comment:
Maybe change it to something like:
```suggestion
Strapi only requires [Node.js](https://nodejs.org). Strapi only supports **LTS versions** of Node.js, the current recommended version to run Strapi is **Node LTS v14**. Other versions of of Node.js may not be compatible with the latest release of Strapi.
```

Because it also supports v12